### PR TITLE
feat: allow logical groups or process view by server to be disabled/enabled

### PR DESCRIPTION
**Pull request by lebris**

### CODE REVIEW

File: src/Services/FeatureChecker.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        $configuration;
+
+    public function __construct(Configuration $configuration)
+    {
+        $this->configuration = $configuration;
+    }
+
+    public function isEnabled($name)
+    {
+        return $this->configuration->readRequired("features/$name/enabled");
```
Comment (Niktux) [inline src/Services/FeatureChecker.php]: No default behavior ?

Comment (lebris) [inline src/Services/FeatureChecker.php]: Hard to say that all features will be enabled or disabled by default.

So default behaviors are handled in the `config/built-in/feature.yml` file.

Comment (Niktux) [inline src/Services/FeatureChecker.php]: I think it's more secure to set these configuration entries optional.

```
 return $this->configuration->read("features/$name/enabled", true);
```

Btw, no BC break with this choice !

Comment (lebris) [inline src/Services/FeatureChecker.php]: Fixed f958425ebefac5618af9372840dcae6b7dfb4141

**Pull request merged**